src/processor/operator/persistent/copy_node.cpp

```diff
@@ -20,7 +21,9 @@ void CopyNodeSharedState::init() {
         pkIndex = std::make_unique<PrimaryKeyIndexBuilder>(indexFName, *pkType);
         // Since hashIndexBuilder doesn't support dynamic rehash, we need to reserve enough number
         // of slots when copying turtle files.
-        pkIndex->bulkReserve(numRows);
+        auto sharedState =
```

Comment:
Is this correct? How do we know this sharedState is `ParquetScanSharedState`. Also the comment above is out-dated.

Reply:
Should be ScanSharedTableFuncState